packages/deces-dataprep/Makefile

```
@@ -248,34 +400,179 @@ backup-push: data-tag backup
+	@ES_BACKUP_NAME=esdata_tmp_${DATAPREP_VERSION}_$$(cat ${DATA_TAG});\
+	if [ -d "${INFRA_PATH}" ]; then\
+		${MAKE} -C ${INFRA_PATH} elasticsearch-repository-backup\
+			REPOSITORY_BUCKET=${REPOSITORY_BUCKET} STORAGE_ACCESS_KEY=${STORAGE_ACCESS_KEY} STORAGE_SECRET_KEY=${STORAGE_SECRET_KEY}\
+			ES_INDEX=${ES_INDEX} ES_BACKUP_NAME=$${ES_BACKUP_NAME} ${MAKEOVERRIDES};\
+	else\
+		${MAKE} -C ${DATAPREP_BACKEND_PATH} elasticsearch-repository-backup-async\
+			REPOSITORY_BUCKET=${REPOSITORY_BUCKET} STORAGE_ACCESS_KEY=${STORAGE_ACCESS_KEY} STORAGE_SECRET_KEY=${STORAGE_SECRET_KEY}\
+			ES_INDEX=${ES_INDEX} ES_BACKUP_NAME=$${ES_BACKUP_NAME} ${MAKEOVERRIDES};\
+	fi
 
 respository-cleanse:
```

checkmake (0.3.2)

[warning] Target body for "respository-cleanse" exceeds allowed length of 5 lines (10).

(maxbodylength)